goss-testing/scripts/ceph-service-status.sh

```diff
@@ -65,7 +65,7 @@ function check_service(){
       fi
       if [[ -n "$osd_id" ]] || [[ -n "$osd" ]]
       then
-        read -r -d "\n" service_unit status < <(pdsh -N -w "$host" podman ps --format json 2>&1 |grep -v "Permanently added"|jq --arg osd "$osd_prefix$osd_id" -r '.[]|select(.Names[]|contains($osd))|.Names[], .State')
+        read -r -d "\n" service_unit status < <(pdsh -N -w "$host" podman ps --format json 2>&1 |grep -v "Permanently added"|jq --arg osd "$osd_prefix$osd_id" -r '.[]|select(.Names[]|endswith($osd))|.Names[], .State')
```

Comment:
do we need a conditional that checks the ceph version here?   basically a 1.4 could have a ceph 16.2.x or 17.2.x cluster.  I know some of this changes between versions, so lets double check this

Reply:
you basically want to make sure all these queries work against 16.2 and 17.2.

Reply:
I ran this on Fanta which is running CSM 1.5 and has Ceph version 17.2.6. I also ran this on Drax which is running CSM 1.4.1 and has Ceph version 16.2.13. I believe this works for both Ceph versions.